.travis.yml

```diff
@@ -1,26 +1,68 @@
 language: python
 
-python:
-  - 3.6
-
-env:
-  - TOX_ENV=py26
-  - TOX_ENV=py27
-  - TOX_ENV=pypy2.7
-  - TOX_ENV=py27-numpy
-  - TOX_ENV=py32
-  - TOX_ENV=py33
-  - TOX_ENV=py34
-  - TOX_ENV=py36
-  - TOX_ENV=py36-numpy
-  - TOX_ENV=pypy3.6
-  - TOX_ENV=docs-py3
+matrix:
+  include:
+    - python: 2.7
+      env:
+        - TOX_ENV=py26
```

Comment:
Time to drop 2.6, perhaps? It's well out of support. I've dropped 2.x altogether in my packages.

Reply:
The plan is, vaguely, to cut a 2.0.0 version and drop Python 2.*